detection_rules/rule.py

```diff
@@ -30,14 +35,32 @@ class RuleMeta(MarshmallowDataclassMixin):
     deprecation_date: Optional[definitions.Date]
 
     # Optional fields
-    beats_version: Optional[definitions.BranchVer]
-    ecs_versions: Optional[List[definitions.BranchVer]]
     comments: Optional[str]
     maturity: Optional[definitions.Maturity]
+    min_stack_version: Optional[definitions.SemVer]
     os_type_list: Optional[List[definitions.OSType]]
     query_schema_validation: Optional[bool]
+    query_validate_master: Optional[bool]
```

Comment:
Not clear what this means. Could we drop it?

Reply:
Could you just do `min_stack_version = "master"` in TOML instead?

Reply:
Remember, the way it goes is to validate _all_ stack versions from the minimum to the most recent. This is a way to specify that, in addition to that, to also validate against master for ECS and beats. See 20 lines down

Reply:
So I think these two things are one and the same.
> all stack versions from the minimum to the most recent.

`packages.yml` should specify what "most recent" means. So that way, in the `main` branch, we can validate against `master`, but in `7.13` we don't.

Reply:
you are correct, I mistyped. Updated my comment